static/js/components/tiles/bar_tile.tsx

```diff
@@ -227,9 +226,24 @@ export const fetchData = async (props: BarTilePropType) => {
   try {
     const statResp = await statPromise;
     const denomResp = await denomPromise;
-
     // Find the most populated places.
     const popPoints: RankingPoint[] = [];
+    // Non-place entities won't have a value for Count_Person.
+    // In this case, make an empty list of popPoints
+    if (_.isEmpty(statResp.data[FILTER_STAT_VAR])) {
+      const entityDcidsSet = new Set<string>();
+      Object.keys(statResp.data).forEach((statVarKey) => {
+        Object.keys(statResp.data[statVarKey]).forEach((entityDcid) => {
+          entityDcidsSet.add(entityDcid);
+        });
+      });
+      entityDcidsSet.forEach((entityDcid) => {
+        popPoints.push({
+          placeDcid: entityDcid,
```

Comment:
At some point (not advocating in this PR) it'll be good to have a cleaner distinction between place and non-place entities.

Reply:
Agreed - we should use the more generic "entity" terminology vs place